plugins/s3_auth/s3_auth.cc

```diff
@@ -685,6 +889,9 @@ TSRemapNewInstance(int argc, char *argv[], void **ih, char * /* errbuf ATS_UNUSE
     {const_cast<char *>("secret_key"), required_argument, nullptr, 's'},
     {const_cast<char *>("version"), required_argument, nullptr, 'v'},
     {const_cast<char *>("virtual_host"), no_argument, nullptr, 'h'},
+    {const_cast<char *>("v4-include-headers"), required_argument, nullptr, 'i'},
+    {const_cast<char *>("v4-exclude-headers"), required_argument, nullptr, 'e'},
```

Comment:
What happens now if someone tries to use the v4 specific configurations with a v2 rule? Will it just accept it, or does it give appropriate warnings errors?